packages/fxa-metrics-processor/src/test/lib/amplitude/transforms/common.spec.ts

```diff
@@ -12,13 +12,13 @@ import {
   mapLocation,
   tee,
   sha256Hmac,
```

Comment:
Everything below (if you are viewing the PR in a browser...not VS Code) is either formatting changes or `marketingOptIn` code that's been removed.